src/main/java/animals/birds/Birds.java

```diff
@@ -1,19 +1,21 @@
-package animals;
+package animals.birds;
 
-public class Birds extends Animals {
+import animals.Animals;
+
+public abstract class Birds extends Animals {
     public Birds(String animalName, String typeOfFood) {
         super(animalName, typeOfFood);
     }
 
     @Override
     public String moveMethod() {
 
-        return ", летает";
+        return "летает";
```

Comment:
Here and in other intermediate classes (Birds, Fish, Mammals) I would prefer if this value was stored in a variable on class level. Easier to maintain and modify.
This relates both to moving and breathing methods